### ISSUE TITLE
templates: update home page UI to correspond wireframe

### ISSUE THREAD

**Issue by pamfilos** [cernopendata/opendata.cern.ch#1352]
Update home page to correspond to the [wireframe UI](https://invenio.mybalsamiq.com/projects/cernopendata-clickableprototype/Home%20page) and introduce new features mused IRL (ex. news ticker/carousel)

**Comment by tiborsimko** [comment]
Don't forget to prominently lead people also to the event display etc. Is there an updated wireframe?

**Comment by daslerr** [comment]
There isn't an updated wireframe, but I talked with @ioannistsanaktsidis IRL about the event displays. Both of the event displays and the histogram viewer will be shown on the homepage in blocks and they'll link to the individual event display and histogram pages. We'll also add a link to the `/visualise` page under the Resources header on the homepage. 

Other homepage updates, for documentation purposes: 
- A link to the Learning Resources will also be added under the current Resources header. 
- The Resources header will be renamed to Explore which has a friendly sound for the public and won't be confused with Learning Resources. 
- We've picked a few placeholder images to use for the articles on the homepage, to show what it could look like. We can consult with the collaborations about real images to use when we finalize article content.
